site/content/docs/main/locations.md

```diff
@@ -178,3 +232,8 @@ velero backup create full-cluster-backup
 [2]: api-types/volumesnapshotlocation.md
 [3]: https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure/blob/main/volumesnapshotlocation.md
 [4]: https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure/blob/main/backupstoragelocation.md
+[5]: /plugins
+[6]: overview-plugins.md
+[7]: https://kubernetes.io/docs/concepts/configuration/secret/
+[8]: #create-a-storage-location-that-uses-unique-credentials
```

Comment:
I dont think the links with the # are working correctly, i can't tell if this is bc there is an issue with the preview, or they will also not work on the live site?

Reply:
Ahhh, I totally thought these worked :woman_facepalming: okay, let me figure out how to fix this!